tests/drivers/test_forecast.py

```diff
@@ -165,3 +166,29 @@ def test_create_directory_structure():
             forecast_obj.create_directory_structure(run_directory, "quit")
         assert pytest_wrapped_e.type == SystemExit
         assert pytest_wrapped_e.value.code == 1
+
+def test_create_forecast_obj():
+    ''' Tests that stage_fixed_files() is copying files from config_obj 
+    are being staged in run directory.'''
+
+    with tempfile.TemporaryDirectory() as run_directory:
+
+        test_yaml = os.path.join(uwtools_file_base,pathlib.Path("../fixtures/expt_dir.yaml"))
+        test_cfg = config.YAMLConfig(test_yaml)
+        forecast_obj = FV3Forecast()
+
+        forecast_obj.create_directory_structure(run_directory)
+
+        files_to_stage = test_cfg.get('static', {})
+
+        for target, file_path in files_to_stage.items():
+            pathlib.Path(file_path).touch()
```

Comment:
I didn't know about `Path.touch()`! Thanks, I need this in a bunch of places!

Reply:
Same here. Nice. :)